test/test_legacy_forbidden.py

```diff
@@ -0,0 +1,76 @@
+"""
+In this file we check that if we use legacy features in universal mode, we
+get the expected compile time errors
+"""
+
+import sys
+import pytest
+from .support import HPyTest, make_hpy_abi_fixture
+
+# this is not strictly correct, we should check whether the actualy compiler
+# is GCC. But for the CI and most cases, it's enough to assume that if we are
+# on linux we are using GCC.
+#
+# We need this because some of the nice compilation errors (such as the ones
+# causes by _HPY_LEGACY) are triggered only by gcc. Would be nice to have them
+# also for other compilers
+ONLY_GCC = pytest.mark.skipif(sys.platform!='linux', reason='GCC only')
```

Comment:
Should that maybe go into `support.py` ?

Reply:
fixed by 4907ee8